### DISCUSSION TITLE
Imaginary part of signal

### DISCUSSION THREAD

**Discussion by ryankong123** [chulminy/CIVE497-CIVE700#121] [General]
Hi Professor,

I was wondering what the imaginary part of a signal represents.

Please advise.

Thanks

**Comment by chulminy** [comment]
The imaginary part of the signal is just considered as a mathematical representation of actual signals.

If you see the complex form of the Fourier series, a complex form is just another way of representing time signals. 

It's just mathematic and you do not read too much on this. 

It's not possible to imagine "imaginary" signals.

**Reply by ryankong123** [reply]
Thanks!

**Reply by chulminy** [reply]
You will have similar doubts when you learn homogenous coordinates. 

Then, you can consider it as a "mathematical" tool like an imaginary signal.
